run/helloworld/spec/e2e_spec.rb

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "open3"
+require "rspec"
+require "rest-client"
+require "securerandom"
+
+describe "E2E tests" do
+  before (:all) do
+    suffix = SecureRandom.hex(15)
+    system("gcloud", "builds", "submit", "--project=#{ENV["GOOGLE_CLOUD_PROJECT"]}", "--config=e2e_test_setup.yaml", "--substitutions=_SUFFIX=#{suffix}")
```

Comment:
Is there a way to make this more readable, maybe on multiple lines?

Reply:
done.